plugins/gradle/src/main/java/org/wildfly/swarm/plugin/gradle/PackageTask.java

```diff
@@ -66,6 +71,23 @@ public void packageForSwarm() throws Exception {
 
         GradleArtifactResolvingHelper resolvingHelper = new GradleArtifactResolvingHelper(project);
         Properties propertiesFromExtension = getPropertiesFromExtension();
+        List<File> moduleDirs = getModuleDirs();
+        if (moduleDirs.isEmpty()) {
```

Comment:
OK, I'm not a skilled Gradle user, but I think that setting `modulesDir` from `build.gradle` doesn't necessarily overwrite the existing value -- it can do both. Right?